package exception;

import java.util.InputMismatchException;
import java.util.Scanner;

public class ExceptionTest {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		System.out.println("정수를 입력하세요 >>>");
		
		int num;
		try {
			num = sc.nextInt();
			System.out.println("입력한 수는 " + num + "입니다.");
		} catch (InputMismatchException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
			System.out.println("정수를 입력하라니까!!!");
		}
		


	}

}
